src/model/wallet.ts

```diff
@@ -257,11 +258,17 @@ export const walletInit = async (
   return { isNew, walletAddress };
 };
 
-export const loadWallet = async (
-  address?: EthereumAddress | undefined,
-  showErrorIfNotLoaded = true,
-  provider?: Provider
-): Promise<null | Wallet | LedgerSigner> => {
+export const loadWallet = async <S extends Screen>({
+  address,
+  showErrorIfNotLoaded = false,
```

Comment:
checking on why the change in default from `true` to `false` for this param

Reply:
Whoops good catch! Let me fix that and the default instances

Reply:
Fixed: https://github.com/rainbow-me/rainbow/pull/5962/commits/a0dbfc61d6f4bd560da1b06c57d14337c8258718